### ISSUE TITLE
"Active reference" not passed to COORDS data sets

### ISSUE THREAD

**Issue by drroe** [Amber-MD/cpptraj#82]
When a COORDS data set is created it creates a separate copy of the associated topology. However, this means that any selection by distance that relies on the COORDS internal topology reference coordinates will fail since these are not ever updated. It may be time to rework the "internal reference" coords of topology files used for distance-based mask selection.

**Comment by drroe** [comment]
For my info: this is the relevant part of the bug as reported by a user:
```
parm pro_com.prmtop
reference wat_box.inpcrd
trajin ../traj/md1.nc
rmsd rms_test :250<:10&@CA,N,C reference out rms_test.dat time 0.002
Error: No reference set for [pro_com.prmtop], cannot select by distance
Error: Could not initialize action [rmsd]
```

**Comment by drroe** [comment]
Fixed by #111
